whitepapers/0013_ccq.md

```diff
@@ -317,7 +317,42 @@ Currently the only supported query type on Solana is `sol_account`.
 
    - The `data_slice_offset` and `data_slice_length` are optional and specify the portion of the account data that should be returned.
 
-   - The `account_list` specifies a list of accounts to be batched into a single call. Each account in the list is a Solana `PublicKey`
+   - The `account_list` specifies a list of accounts to be batched into a single query. Each account in the list is a Solana `PublicKey`
+
+2. sol_pda (query type 5) - this query is used to ready data for one or more accounts on Solana based on their Program Derived Addresses.
+
+   ```go
+   u32         commitment_len
+   []byte      commitment
+   u64         min_context_slot
+   u64         data_slice_offset
+   u64         data_slice_length
+   u8          num_pdas
+   []byte      pda_list
```

Comment:
`[][]byte`?

Reply:
Changed it to `[]PdaList` which is defined below.